app/scheduler/scheduler.py

```diff
@@ -6,30 +6,35 @@
 from astropy.time import Time, TimeDelta
```

Comment:
Kind of weird to have this doubly nested package name, no?

Reply:
Yes, but this should be scrapped and change instead from a Builder